SecretManagement.KeePass.Extension/Tests/Test-SecretVault.Tests.ps1

```diff
@@ -76,20 +75,61 @@ InModuleScope -ModuleName 'SecretManagement.KeePass.Extension' {
             }
```

Comment:
-Throw should be relatively explicit to the expected exception, in this case: | Should -Throw 'Cannot find a variable with the name*'